WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -1,5 +1,12 @@
 #import <Foundation/Foundation.h>
 
+typedef enum
+{
+    ServiceRemoteRESTApiVersion_1_1 = 1001,
+    ServiceRemoteRESTApiVersion_1_2 = 1002,
+    ServiceRemoteRESTApiVersionDefault = ServiceRemoteRESTApiVersion_1_1,
+} ServiceRemoteRESTApiVersion;
```

Comment:
I originally had constant strings, but it felt weird to allow anything in (ie: the user could then pass `@"hello world"` as the version number).